log4j-core-test/src/test/java/org/apache/logging/log4j/core/config/xml/XmlConfigurationPropsTest.java

```diff
@@ -72,9 +72,17 @@ void testNoProps(final Configuration config) {
     }
 
     @Test
+    @SetTestProperty(key = StatusLogger.DEFAULT_STATUS_LISTENER_LEVEL, value = "INFO")
     @SetTestProperty(key = Constants.LOG4J_DEFAULT_STATUS_LEVEL, value = "WARN")
```

Comment:
In order to test `DEFAULT_` has precedence over `LOG4J_`, shouldn't the latter has a more specific level than `WARN`? That is, shouldn't `LOG4J_` level be, say, `DEBUG`?